mlrun/db/httpdb.py

```diff
@@ -1701,7 +1701,7 @@ def list_feature_sets(
 
     def store_feature_set(
         self,
-        feature_set: Union[dict, schemas.FeatureSet],
+        feature_set: Union[dict, schemas.FeatureSet, FeatureSet],
```

Comment:
I assume this change is done to enable the test contained in this PR. If we enable this in store, it makes sense to do the same in `create_feature_set` as well.

Reply:
true. It was also inspired by `store_project` which has a similar implementation and gives better robustness.